client/components/navbar.js

```diff
@@ -1,17 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
are we using useState in the Navbar functional component?

Reply:
i tried to use it , but didn't work the way i wanted